Runtime/Scripts/AudioStreamTrack.cs

```diff
@@ -319,23 +186,23 @@ public AudioStreamTrack(AudioSource source)
                 throw new ArgumentNullException("AudioSource argument is null.");
             if (source.clip == null)
```

Comment:
@aet 
I may have misunderstood your comment.
Did you mean we can remove this line?

Reply:
No misunderstanding, glad to hear AudioCustomFilter works for both cases. I only realized later it was using a different SetData implementation!

I had the clip check removed before, but thought maybe it was there for a reason. I'm all forward removing it if no harm done

Reply:
I will make another PR for this fix.